spec/rspec/support/object_formatter_spec.rb

```diff
@@ -95,6 +95,18 @@ def with_date_loaded
         end
       end
 
+      context 'given a delegator' do
+        let(:object) { Object.new }
+        let(:delegator) do
+          require "delegate"
+          SimpleDelegator.new(object)
+        end
+
+        it 'includes the delegator class in the description' do
+          expect(ObjectFormatter.format(delegator)).to eq "#<SimpleDelegator(#{object.inspect})>"
+        end
+      end
```

Comment:
It would be nice to also have a spec for the case where `Delegator` is not defined, since the `defined?(::Delegator)` check is essential but not covered by any test.  You can simulate it not being defined by using `hide_const("Delegator")` in a spec.